integration/src/test/java/edu/unc/lib/boxc/integration/web/access/EndpointIT.java

```diff
@@ -86,20 +86,20 @@ public class EndpointIT {
     protected AdminUnit adminUnit1;
 
     public void createDefaultObjects() throws Exception {
-        adminUnit1 = adminUnitFactory.createAdminUnit(Map.of("title", "Object1"));
-        var adminUnit2 = adminUnitFactory.createAdminUnit(Map.of("title", "Object2"));
+        adminUnit1 = adminUnitFactory.createAdminUnit(Map.of("title", " Admin Object1"));
```

Comment:
I think it gets trimmed off at some point, but were the leading spaces on a number of these titles intentional? If not, it might be good to remove them to avoid confusion.

Reply:
nope, whoops